lib/common.sh

```diff
@@ -625,6 +625,24 @@ common::sha () {
   which shasum >/dev/null 2>&1 && shasum -a$algo $file | awk '{print $1}'
 }
 
+# Check for and source security layer
```

Comment:
"security layer" is pretty vague and unhelpful. "extensions" or something is a little clearer.